Explorer/Assets/Scripts/SceneRuntime/SceneRuntimeImpl.cs

```diff
@@ -99,5 +106,8 @@ public void Dispose()
             engineApi?.Dispose();
             engine.Dispose();
         }
+
+        public ITypedArray<byte> CreateUint8Array(int length) =>
```

Comment:
I wonder if this buffer can be re-used, much like it has been previously pooled. One to discuss with @mikhail-dcl when he is back!